tools/minHash/.shed.yml

```diff
@@ -0,0 +1,17 @@
+name: minHash
+owner: joachimwolff
+description: A tool to compute the nearest neighbors in a high dimensional sparse dataset.
+
+long_description: |
+  MinHash is an approximate nearest neighbors search algorithm for high dimensional sparse datasets. It computes first a candidate set with MinHash and after the 
+  exact neighbors with the euclidean distance or cosine similarity.
+
+categories:
+  - Machine Learning
```

Comment:
Given this [list](https://toolshed.g2.bx.psu.edu/repository/browse_categories), the answer is no. It does not fit in any category listed there.